src/ui/tests/hw-19/task-1.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Open the https://the-internet.herokuapp.com/ website
+// Locate and click the Dynamic Controls link
+// Validate the checkbox
+// Click the "Remove" button
+// Wait for the checkbox to be removed
+// Verify the presence of the "Add" button
+// Validate the "It's gone!" text
+// Click the "Add" button
+// Wait for the checkbox to appear
+// Validate the "It's back!" text
+
+import { test, expect } from '@playwright/test';
+
+test('Dynamic controls', async ({page}) => {
+    const dynamicControlsLink = page.getByRole('link', { name: 'Dynamic Controls' });
+    const checkbox = page.getByRole('checkbox');
+    const addButton = page.getByRole('button', { name: 'Add' });
+    const removeButton = page.getByRole('button', { name: 'Remove' });
+    const itsGoneText = page.getByText('It\'s gone!');
```

Comment:
Метод `getByText` возвращает локатор, а по названию переменной понимается, как обычный текст в ней
```suggestion
    const itsGoneLocator = page.getByText('It\'s gone!'); // itsGoneElement | itsGoneMessage
```
[getByText](https://playwright.dev/docs/api/class-page#page-get-by-text)